config/templates/minishiftDeleteDashboard.sh

```diff
@@ -0,0 +1,20 @@
+#bin/bash 
+
+if [ "$#" -eq 0 ]
```

Comment:
Don't like the bash style here, I'd rather this be
```
if [ "$#" -eq 0 ] ; then
  ...
elif
```
for example, otherwise there's just so much pointless whitespace